pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -71,18 +82,45 @@ func (q *QueryReconciler) Reconcile(ctx context.Context, req *ctrl.Request) (ctr
 		return res, err
 	}
 
-	return ctrl.Result{}, q.runListRestorePoints(ctx, req)
+	finder := iter.MakeSubclusterFinder(q.VRec.Client, q.Vdb)
+	pods, err := finder.FindPods(ctx, iter.FindExisting)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+
+	// find a pod to execute the vclusterops API
+	var podIP string
+	for i := range pods.Items {
+		if i != names.GetNMAContainerIndex() {
+			continue
+		}
+		pod := &pods.Items[i]
+		podIP = pod.Status.PodIP
+		break
+	}
+
+	// setup dispatcher for vclusterops API
+	passwd, err := q.GetSuperuserPassword(ctx, q.Log, q.Vdb)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	prunner := cmds.MakeClusterPodRunner(q.Log, q.VRec.Cfg, q.Vdb.GetVerticaUser(), passwd)
```

Comment:
We don't need a pod runner since the dispatcher is going to be vclusterOps only.